Source/compositorclient/Wayland/Westeros.cpp

```diff
@@ -566,7 +566,7 @@ namespace Wayland {
     {
         ASSERT (order <= std::numeric_limits<uint8_t>::max());
```

Comment:
We upgraded the zOrder we pass from the Compositor, to u uint16_t (used to be uint8_t indeed). So could you upgrade it to a uint16_t allows for 65535 layers, i.s.o. 256 :-)

Reply:
Sure will do

Reply:
The smaller difference the fraction is not working with ZOrder settings, it still behaves like 1.0 value. So it seems it supports only upto 255 layers